app/helpers/episodes_helper.rb

```diff
@@ -67,6 +67,20 @@ def episode_media_status(episode)
     end
   end
 
+  def episode_transcript_status(episode)
+    if upload_problem?(episode.transcript)
+      "error"
+    elsif upload_processing?(episode.transcript)
+      "processing"
+    elsif episode.transcript.present?
+      "complete"
+    elsif episode.published_at.present?
+      "incomplete-published"
```

Comment:
I think we can omit these badges for transcripts - it's optional, unlike the media files.